sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py

```diff
@@ -93,7 +93,7 @@ def test_client_repr(LCClient):
     Repr check
     """
     output = str(LCClient)
-    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClien'
+    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClient'
```

Comment:
Unsure with the :50 it might end early?

Reply:
Oh I should add :51 to make sure its ok.